cmd/cluster-node-tuning-operator/main.go

```diff
@@ -194,6 +204,46 @@ func operatorRun() {
 		if err = (&performancev2.PerformanceProfile{}).SetupWebhookWithManager(mgr); err != nil {
 			klog.Exitf("unable to create PerformanceProfile v2 webhook: %v", err)
 		}
+	} else {
+		// Hypershift configuration
```

Comment:
Nit: Shouldn't this be `HyperShift`?  If so, let's be consistent everywhere.

Reply:
Done